sysutils/tmux/Portfile

```diff
@@ -9,8 +9,8 @@ if {${subport} eq ${name}} {
     conflicts       tmux-devel
 }
 subport tmux-devel {
-    github.setup    tmux tmux 47712fc113fbc602e27aa1f24e5ada3c2f061a94
-    version         20190719-[string range ${github.version} 0 6]
+    github.setup    tmux tmux 24ab1bc714b80d957b53685635a44dc0a89c032b
+    version         20190919-[string range ${github.version} 0 6]
     revision        1
```

Comment:
The `revision` needs to be set back to 0 when increasing the `version`.

Reply:
thanks, yes, you are right. I missed this, because somebody else changed the file in the meantime. My process is automated but didn't account for revision changes by somebody else.

Reply:
Done